core/src/main/java/com/netflix/conductor/core/events/DefaultEventQueueManager.java

```diff
@@ -158,6 +158,12 @@ public void refreshEventQueues() {
     }
 
     private void listen(ObservableQueue queue) {
-        queue.observe().subscribe((Message msg) -> defaultEventProcessor.handle(queue, msg));
+        LOGGER.info("Listening to observable queue: {}", queue.getType());
+        if (queue.getType().equals("kafka")) {
+            LOGGER.info("Using new queue handler for Kafka");
+            queue.observe((Message msg) -> defaultEventProcessor.handle(queue, msg));
```

Comment:
Use a different approach to handle event messages for our kafka event queue implementation. Since we are not handling messages sequentially with observables, just passing in the handler function.